src/ad-tracking/compilers/slot-tracking-compiler.ts

```diff
@@ -53,9 +53,11 @@ export const slotTrackingCompiler = ({ data, slot }: CompilerPartial): CompilerP
 			page_width:
 				(window.document.body.scrollWidth && window.document.body.scrollWidth.toString()) || '',
 			ppid: targetingData.ppid,
+			pa_available: targetingService.get('pa_available'),
```

Comment:
and use ternary operator here similar to `is_uap`
```suggestion
			pa_available: targetingService.get('pa_available') ? 1 : 0,
```
we will send less bytes over the internet and sending boolean as string is kinda code smell to me